### DISCUSSION TITLE
[日常][委托]有大佬能够帮我看看吗？

### DISCUSSION THREAD

**Discussion by 1155993355** [share121/inter-knot#5774] [公告]
不知道我这么多母带，这么多菲林，能不能把雅娶回家？
![Screenshot_2024-12-15-19-28-14-17_723a72aa3a9195ba0d239017b796a750](https://github.com/user-attachments/assets/9e06546d-6178-43a6-a5df-98fb7b030420)

**Comment by 1155993355** [comment]
我手上没有一个冰c，你们说我是等雅出来抽，还是再忍一阵子，静候佳音

**Reply by woaixier** [reply]
你爱吗，爱的话就不需要理由

**Comment by yanER582136479** [comment]
抽啊，雅的强度太高了，操作也简单，抽个0+1就够爽很久了，当然歪了就静候佳音……
